public/components/visualization/text2vega.ts

```diff
@@ -63,6 +63,9 @@ export class Text2Vega {
                 )
                 // eslint-disable-next-line @typescript-eslint/no-explicit-any
                 .toPromise<any>();
+              if (res.rawResponse.total === 0) {
+                throw new Error(`There is no result with the generated query: '${value.ppl}'.`);
+              }
```

Comment:
would there a place to show this error on page?

Reply:
Yes, in [text2viz.tsx](https://github.com/opensearch-project/dashboards-assistant/pull/330/files#diff-44cdcb0c0e12c1b4082203f1a1030c5a257a8cec28587574f3e43811eadf1dbb) line 110